spec/menu_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'menu'
+
+describe Menu do
+
+  let(:dish_example) { {number: 1, dish: 'Chicken Biryani', price: 8.99} }
+
+  describe '#list_dishes' do
+    it "should show the available dishes in the menu" do
+      expect(subject.list_dishes).to include dish_example
```

Comment:
I think you are testing state rather than behaviour. Maybe better to test that the return value is a string listing the dishes with prices, like ```"Chicken Biryani (£8.99), etc."